cdap-e2e-tests/src/e2e-test/features/controlcenter/ControlCenter.feature

```diff
@@ -0,0 +1,102 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+@Controlcenter
+
+Feature: Controlcenter - Validate control center page flow
+
+  Scenario: Verify user is able to click the control center tab and successfully navigates to control center page
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Verify that the user is navigated to control center page successfully
+
+  @BQ_INSERT_INT_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: Verify that user is able to create a pipeline and then validate the presence of created pipeline in control center.
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Click on the Plus Green Button to import the pipelines
```

Comment:
Can we change it to Add Entity Button ?

Reply:
We are not importing piplines , we are adding new entities here by creating , importing and uploading . So change the scenario text : Click on the Add Entity Button inside Control Center

Reply:
step is there already in the framework